app/models/comment.rb

```diff
@@ -47,6 +47,10 @@ class Comment < ActiveRecord::Base
 
   after_destroy do
     self.parent.update_comments_counter
+    participation = Participation.where({ 
+      target_id: self.commentable_id, author_id: self.author_id
+    }).first()
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.<br>Do not use parentheses for method calls with no arguments.